resources/c++/enums_and_unions.cpp

```diff
@@ -0,0 +1,69 @@
+/*
+ * Unscoped enums:
+ */
+enum unscoped_enum { Kartoffel, Karotte, Kohlrabi, Krautsalat };
```

Comment:
I prefer English examples please :-)

Reply:
I thought that would be more funny compared to "entry1, entry2, entry3" or something. Can't we have some nice things like Kartoffeln? :D

Reply:
Next example will be french or italian then? :D 
But I can change that of course

Reply:
Done